scripts/data_transfer/move_uploaded_snpchip_plink.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Copy Plink files from main-upload to main,
+# and transfer a subset of 20 samples to test.
+
+set -ex
+
+# install Plink
+micromamba install -y --prefix $MAMBA_ROOT_PREFIX -y -c bioconda -c conda-forge plink
+
+SAMPLES_FILE="samples_to_keep.txt"
+
+cat > ${SAMPLES_FILE} << EOF
```

Comment:
What do these numbers mean? Is this the sample sample subset that we use for WGS joint calling?

Reply:
Those are the SNPchip Family + Individual IDs for those 20 samples in mt4. That's how Plink subsets by sample.

Reply:
Could you add a comment in the code describing that, please? (In general that's preferable to a response in a review comment, as the next person looking at this code might have a similar question (and won't review PR comments).)

Reply:
Ah, okay, will do!